intranet/apps/dataimport/management/commands/import_users.py

```diff
@@ -0,0 +1,82 @@
+import json
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+
+class Command(BaseCommand):
+    help = "Import Fake People"
+
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument("data_fname")
+
+    def clean(self, name):
+        return name.replace(" ", "").replace("-", "").replace("_", "").replace("'", "")
+
+    def handle(self, *args, **kwargs):
+        with open(kwargs["data_fname"]) as f:
+            data = json.load(f)
+
+        counselor_data = data["counselors"]
```

Comment:
I don't see assigning `counselor_data` as the most clean implementation.
I would just iterate on `data["counselors"]`.